tests/phpunit/tests/test-install.php

```diff
@@ -63,20 +67,22 @@ function test_uninstall() {
 
 		update_post_meta( $item_id, '_pll_menu_item', array() );
 
+		$this->assertFalse( defined( 'PLL_REMOVE_ALL_DATA' ), 'The constant PLL_REMOVE_ALL_DATA must not be defined for this test'  );
+
 		if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
 			define( 'WP_UNINSTALL_PLUGIN', true );
 		}
 
 		include_once dirname( dirname( dirname( dirname( __FILE__ ) ) ) ) . '/uninstall.php';
 		new PLL_Uninstall();
 
-		// Option Uninstall = false => nothing deleted
+		// Constant PLL_REMOVE_ALL_DATA undefined => nothing deleted
 		$options = get_option( 'polylang' );
 		$this->assertNotEmpty( $options );
 
-		// Option Uninstall = true
-		$options['uninstall'] = 1;
-		update_option( 'polylang', $options );
+		// Constant PLL_REMOVE_ALL_DATA = true
+		define( 'PLL_REMOVE_ALL_DATA', true );
+
 		new PLL_Uninstall(); // Fire uninstall process again
 
 		// No options
```

Comment:
The following assertions would be better as part of another test, as you have two opposite cases here:

- The case were we **don't** want to remove the data
- The case were we want to remove our data

Each of these cases might be working independently from the other, so two tests would provide better information here.